src/bot/say-hello.js

```diff
@@ -7,24 +7,23 @@ export function sayHello() {
     const messageWords = text.toLowerCase().split(' ')
 
     if (!messageWords.some(v => hellos.includes(v))) {
-      return { ...messageContext }
+      return
     }
 
     const mentionSomeone = messageWords.filter(w => w.startsWith('@')).length > 0
     const mentionSanpe = mentionSomeone && messageWords.includes(`@${process.env.BOT_USERNAME.toLowerCase()}`)
 
     if (mentionSomeone && !mentionSanpe) {
       // mention
-      return { ...messageContext }
+      return
```

Comment:
If the fulfilment is by default set as true, In this particular case, you don't want to fulfil with this handler in this case. So it should return a fulfilment as false here